src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -111,33 +117,64 @@ public function create(string $resourceClass, string $name, array $options = [])
                     $iri = self::SCHEMA_MAPPED_CONSTRAINTS[\get_class($constraint)] ?? null;
                 }
 
-                if (null !== $required && null !== $iri) {
-                    break 2;
+                foreach ($this->restrictionsMetadata as $restrictionMetadata) {
+                    if ($restrictionMetadata->supports($constraint, $propertyMetadata)) {
+                        $restrictions[] = $restrictionMetadata->create($constraint, $propertyMetadata);
+                    }
                 }
             }
         }
 
-        return $propertyMetadata->withIri($iri)->withRequired($required ?? false);
+        $propertyMetadata = $propertyMetadata->withIri($iri)->withRequired($required ?? false);
+
+        if (!empty($restrictions)) {
+            if (null === $schema) {
+                $schema = [];
+            }
+
+            $schema += array_merge(...$restrictions);
```

Comment:
Do we really need this spread operator / array_merge?

Reply:
Yes we must  merge all restrictions into one schema